cms/jinja2/templates/base.html

```diff
@@ -8,72 +8,76 @@
 {% endset %}
 
 {# fmt:off #}
-{% set languages = languages | default([
-    {
-        "url": "#",
-        "isoCode": "en",
-        "text": "English",
-        "current": true
-    },
-    {
-        "url": "#",
-        "isoCode": "cy",
-        "text": "Cymraeg",
-        "current": false
-    }
-])
-%}
+{% set languages=translation_urls() %}
 
 {% if page %}
     {% set body_class = ["template-", page.get_verbose_name()| lower | replace(" ", "-")] | join %}
 {% else %}
     {% set body_class = '' %}
 {% endif %}
 
+{# Untranslated pages will have aliases to the original page #}
```

Comment:
```suggestion
{#- Untranslated pages will have aliases to the original page -#}
```

~but also, is that correct with simple translation?~ this is true because we use [`WAGTAILSIMPLETRANSLATION_SYNC_PAGE_TREE`](https://docs.wagtail.org/en/stable/reference/contrib/simple_translation.html#page-tree-synchronization)

Reply:
Addressed in [99db1b3](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/99db1b3e24591fccea192c4a305f5199cb698de7)